package.json

```diff
@@ -25,15 +25,14 @@
     "access": "public"
   },
   "browserslist": [
-    "last 2 versions",
-    "ie 11"
+    "extends @edx/browserslist-config"
   ],
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-component-footer": "11.6.0",
     "@edx/frontend-component-header": "3.5.0",
```

Comment:
Are we un-pinning header / footer as part of this? Or follow up

Reply:
I wanted to unpin it as a follow-up, because its managed in a different repo and it's easier to coordinate